rules/azure-resources-creating/rule.md

```diff
@@ -54,53 +52,34 @@ Some people half solve the problem by manually creating and saving the script. T
 
 So if you aren't manually creating your Azure resources, what options do you have?
 
-### Option A: Terraform
-
-https://www.terraform.io/docs/providers/azurerm/index.html
-
-* It’s a great tool
-* Free for up to 5 users with limited features
-* Not recommended because:
-    * Pulumi is better
-    * Proprietary ‘HCL’ (Hashicorp Configuration Language) which is as bad as YAML
-
-### Option B: Ansible
-
-[https://www.ansible.com](https://www.ansible.com/)
-
-* Proprietary product owned by RedHat
-* First red flag – ‘Contact us for pricing’ – a toxic warning sign of their lack of transparency
-
-### Option C: Bicep by Microsoft
+### Option A: Bicep by Microsoft (Recommended)
 
 [https://github.com/Azure/bicep](https://github.com/Azure/bicep)
 
-* Experimental
-* Not a huge step forward from ARM templates
-* But is one to watch
+* Is free and fully supported by Microsoft
+* * Has 'dotnet' command line integration
```

Comment:
I think you're referring to `az` instead of `dotnet`

https://docs.microsoft.com/en-us/cli/azure/bicep?view=azure-cli-latest

Reply:
maybe include that link as part of the rule ;)